File: python/2437.number-of-valid-clock-times.py
class Solution:
    def countTime(self, time: str) -> int:
        res = 0
        for hours in range(0, 24):
            for minutes in range(0, 60):
                str_hours = str(hours)
                if hours < 10:
                    str_hours = "0"+str_hours
                str_min = str(minutes)
                if minutes < 10:
                    str_min = "0"+str(minutes)
                try_time = str_hours+":"+str_min
                same = True
                for i in range(len(time)):
                    if time[i] == '?':
                        continue;
                    if time[i] != try_time[i]:
                        same = False
                res += same
        return res

File: cpp/2437.number-of-valid-clock-times.cpp
class Solution {
public:
    int countTime(string time) {
        int res =0 ;
	for(int i =0  ;i  < 24; i++){
		for(int j = 0 ; j < 60; j++){
			string h = to_string(i), m = to_string(j);
			if(i<10)h = "0"+h;
			if(j<10)m = "0"+m;
			bool same = true;
			string time2 = h+":"+m;
			for(int k = 0; k < time.size(); k++){
				if(time[k] == '?')continue;
				if(time2[k] != time[k]) same=false;
			}
			res += same;
		}
	}
	return res;	
    }
};
